src/test/e2e/api/admin/user/pat.e2e.test.ts

```diff
@@ -149,10 +152,72 @@ test('should fail creation of PAT with passed expiry', async () => {
     await request
         .post('/api/admin/user/tokens')
         .send({
+            description: 'my expired pat',
             expiresAt: yesterday,
         } as IPat)
         .set('Content-Type', 'application/json')
-        .expect(500);
+        .expect(400);
+});
+
+test('should fail creation of PAT without a description', async () => {
+    await app.request
+        .post('/api/admin/user/tokens')
+        .send({
+            expiresAt: tomorrow,
+        } as IPat)
+        .set('Content-Type', 'application/json')
+        .expect(400);
+});
+
+test('should fail creation of PAT with a description that already exists', async () => {
```

Comment:
I think this could be a little more explicit about the context

```suggestion
test('should fail creation of PAT with a description that already exists for the current user', async () => {
```